arviz/plots/dotplot.py

```diff
@@ -0,0 +1,229 @@
+import numpy as np
+
+from ..rcparams import rcParams
+from .plot_utils import get_plotting_function
+
+
+def plot_dot(
+    values=None,
```

Comment:
shouldn't values be a required argument instead of having a default of none and raise an error if it's none?